modules/cli/src/main/scala/scala/cli/commands/config/Config.scala

```diff
@@ -88,11 +99,19 @@ object Config extends ScalaCommand[ConfigOptions] {
             val pgpSecretBase64 = pgpSecret0.map(Base64.getEncoder.encodeToString)
 
             db.set(secKeyEntry, PasswordOption.Value(pgpSecretBase64.toConfig))
-            db.set(secKeyPasswordEntry, PasswordOption.Value(password.toConfig))
             db.set(pubKeyEntry, PasswordOption.Value(pgpPublic.toConfig))
             db.save(directories.dbPath.toNIO)
               .wrapConfigException
               .orExit(logger)
+
+            logger.message("PGP keychains written to config")
+            passwordOpt.foreach { password =>
+              logger.message(
+                s"""Password: ${password.value}
+                   |Don't lose it!
+                   |""".stripMargin
+              )
```

Comment:
I think this should be printed only if `options.pgpPassword.contains("random")`...
Also, `logger.message` means that this might not get printed when verbosity is decreased, while it's actually crucial to print the random password, right?
so maybe `println` would be better 🤔

Reply:
I had in mind adding some assurance for the user that the password that they provided has been interpreted as intended and has not been misspelled.

True, it should be `println` here.

Reply:
> I think this should be printed only if options.pgpPassword.contains("random")...
 
I agreed, we should print password only if it is generated by us.